.azurepipelines/publish-maven-1ES.yml

```diff
@@ -74,14 +74,43 @@ extends:
           inputs:
             tasks: publish
             publishJUnitResults: false
-        - task: SFP.release-tasks.custom-build-release-task.EsrpRelease@4
-          displayName: ESRP Release
+      
+      - job: extractAdditonalData 
+        dependsOn: [Phase_1]
+        displayName: 'Extract Additional Data'
+        variables:
+          ARCHIVE_NAME: 'AppCenter-SDK-Android-'
+        templateContext:
+          outputs:
+          - output: pipelineArtifact
+            displayName: 'Publish additional data'
+            targetPath: '$(Build.ArtifactStagingDirectory)/additionalData'
+            artifactName: additionalData
+        steps:
+        - task: DownloadPipelineArtifact@2
+          displayName: Download Build Artifacts
+          inputs:
+            artifactName: Release
+            targetPath: '$(Build.ArtifactStagingDirectory)/Release'
+        - task: Bash@3
+          displayName: 'Extract Version info'
+          inputs:
+            targetType: 'inline'
+            script: |
+              VERSION=$(grep "versionName = '" versions.gradle | awk -F "[']" '{print $2}')
+              mkdir -p $(Build.ArtifactStagingDirectory)/additionalData
+
+              FULL_ARCHIVE_NAME="${ARCHIVE_NAME}${VERSION}"
+              echo "##vso[task.setvariable variable=ARCHIVE_NAME]$FULL_ARCHIVE_NAME"
+              echo $VERSION > $(Build.ArtifactStagingDirectory)/additionalData/version
+        - task: Bash@3
+          displayName: 'Zip all AAR files'
           inputs:
-            ConnectedServiceName: ESRP Release
-            FolderLocation: $(Build.ArtifactStagingDirectory)/com/microsoft
-            Owners: lucen@microsoft.com
-            Approvers: lucen@microsoft.com
-          enabled: true
+            targetType: 'inline'

```

Comment:
Initially that was my idea but to do that I'd need to CopyFiles task and ArchiveFIles task to archive those  files because i don't need to archive all of them. Seemed redundant to have 2 tasks such scenario and unnecessary copying of files.